native/app/components/ListPanel.js

```diff
@@ -132,7 +140,7 @@ class ListPanel extends React.Component {
 
         // Show snackbar
         this._showSnackbar({
-          text: 'Note deleted!',
+          text: `Deleted ${ deletedNote.length } note${ deletedNote.length > 1 ? 's' : '' }`,
```

Comment:
This needs to be properly translated

Reply:
@flodolo How would we do this (in web extensions for example)

Reply:
in Webextensions there's no way to do it, because WE don't support plural (meh…).

So, you would need to do something like "Delete notes ($number)".

In Fluent, you could do it with a proper plural
https://projectfluent.org/fluent/guide/selectors.html